src/App.jsx

```diff
@@ -1,35 +1,28 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { Navigate, Route, Routes } from "react-router-dom";
+import Header from "./components/Header";
+import Home from "./pages/Home";
+import MyArticle from "./pages/MyArticle";
+import Profile from "./pages/Profile";
+import Article from "./pages/Article";
+import NoPage from "./pages/NoPage";
+import "./reset.css";
+import "./App.css";
 
 function App() {
-  const [count, setCount] = useState(0)
-
   return (
     <>
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.jsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
+      <Header />
+      <main>
+        <Routes>
+          <Route index path="/home" element={<Home />} />
+          <Route path="/home/:article_id" element={<Article />} />
```

Comment:
you might want to have this url be more reflective of the url we navigate from, for example if /myarticles is where all articles are rendering, then maybe /myarticles/:article_id would be more appropriate for a single article as we've just come from there, if that makes sense :)

Also technically we don't need this at all anyway currently as it's not relevant to the current ticket.

Reply:
myarticles will be the url to see the articles the user posted, but it seems ambigous so will think of another name :)